org.hl7.fhir.utilities/src/main/java/org/hl7/fhir/utilities/npm/NpmPackageIndexBuilder.java

```diff
@@ -45,7 +45,7 @@ public void start(String filename) {
         new File(filename).delete();
         conn = DriverManager.getConnection("jdbc:sqlite:"+filename); 

```

Comment:
So, now that I get a longer look at this; are we not locking ourselves into an implementation with this? Are we then forcing ourselves to use sqlite forever, or until we need to break things by switching?

Reply:
what else could we use? For this, I don't know of anything, so we leave this as it is, I think.